__tests__/spec/errors.js

```diff
@@ -12,17 +13,44 @@ process.env.IS_INTEGRATION_TEST = 'true'
 
 let kitRoutesApi
 
+const getPageTitle = html => {
+  const match = html.match(/<title>((.|\n|\r)*)<\/title>/)
+  if (match) return match[1].trim()
+}
+const getH1 = html => {
+  const match = html.match(/<h1.*>((.|\n|\r)*)<\/h1>/)
+  if (match) return match[1].trim()
+}
+const getFirstParagraph = html => {
+  const match = html.match(/<p .*>((.|\n|\r)*)<\/p>/)
+  if (match) return match[1].trim()
+}
+
 describe('error handling', () => {
   let testRouter
 
   beforeEach(() => {
     jest.resetModules()
 
+    const pluginsApi = require('../../lib/plugins/plugins')
+    const sessionApi = require('../../lib/session')
+    const originalGetAppViews = pluginsApi.getAppViews
+
     kitRoutesApi = require('../../lib/routes/api')
     kitRoutesApi.resetState()
     testRouter = kitRoutesApi.external.setupRouter()
 
     jest.spyOn(global.console, 'error').mockImplementation()
+    jest.spyOn(sessionApi, 'getSessionMiddleware').mockReturnValue((req, res, next) => {
+      req.session = {}
+      next()
+    })
+    jest.spyOn(pluginsApi, 'getAppViews').mockImplementation(function () {
+      const output = originalGetAppViews(...arguments)
```

Comment:
This is fine, but I think I would use the array spread operator here as it's clearer for me to see the order the array is constructed:

```
jest.spyOn(pluginsApi, 'getAppViews').mockImplementation(() => [
  path.join(__dirname, '..', 'fixtures', 'mockNunjucksIncludes'),
  path.join(__dirname, '..', '..', 'lib', 'nunjucks'),
  ...originalGetAppViews()
])
```